css/style.css

```diff
@@ -85,3 +85,334 @@ header {
 .link {
   text-decoration: none;
 }
+
+.card {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  justify-items: center;
+  margin: 30px;
+  gap: 30px;
+}
+
+.card-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-around;
+  width: 329px;
+  height: 555px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+}
+
+.card-section-text {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+}
+
+.heading-title {
+  color: #172b4d;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.card-sub-container {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+}
+
+.title-canopy {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.title-developer {
+  font-family: "Poppins", sans-serif;
+  color: #7a869a;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.title-year {
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.desciption-p-mobile {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.pro-language {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 15px;
+  list-style-type: none;
+}
+
+.html-1 {
+  background: #ebebff;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 19px;
+  padding: 4px 12px;
+  color: #6070ff;
+  letter-spacing: 0.03;
+}
+
+.btn-see-project {
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  width: 124px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 5px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  background-color: #fff;
+}
+
+.about-section {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 21px;
+  gap: 24px;
+  border-radius: 0 80px 0 0;
+}
+
+.about-section-inside {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 50px;
+  gap: 12px;
+}
+
+.about-heading {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  background-color: #fff;
+  width: 327px;
+  height: 101px;
+  display: flex;
+  align-items: center;
+  padding-bottom: 12px;
+}
```

Comment:
- [ ] Kindly make sure your `About Myself` title have the same color as the one on the Figma design :wink: 

  - ### YOUR DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/212263700-1e7aea05-8c49-4534-9c40-6e05bbf37de2.png)

  - ### FIGMA DESIGN
  
   ![image](https://user-images.githubusercontent.com/87197412/212263813-564a5e90-1522-4959-94ec-b9544ae1b94a.png)